jdaviz/configs/default/plugins/tools.py

```diff
@@ -34,8 +31,7 @@ def activate(self):
             self._mark = PluginLine(self._profile_viewer, visible=False)
             self._profile_viewer.figure.marks = self._profile_viewer.figure.marks + [self._mark, ]
         # Store these so we can revert to previous user-set zoom after preview view
-        pv_state = self._profile_viewer.state
-        self._previous_bounds = [pv_state.x_min, pv_state.x_max, pv_state.y_min, pv_state.y_max]
+        self._previous_bounds = self._profile_viewer.get_limits()
```

Comment:
I thought about having `self._previous_bounds` only keeping Y bounds but then decided not to, in case downstream somehow needs the X bounds too.